src/client.rs

```diff
@@ -384,6 +387,21 @@ pub fn generate_client_stub(
                         )?;
                         writeln!(out, "                .unwrap_lite())")?;
                     }
+                    syntax::Error::ServerDeath => {
+                        writeln!(out, "            assert!(len == 0);")?;
```

Comment:
nit: this assert is checking server behavior and is probably not necessary -- it'll be adding code to clients. (I notice CLike has a similar one.)